errors/abci.go

```diff
@@ -42,9 +42,8 @@ func debugErrEncoder(err error) string {
 	return fmt.Sprintf("%+v", err)
 }
 
-// The defaultErrEncoder applies Redact on the error before encoding it with its internal error message.
 func defaultErrEncoder(err error) string {
-	return Redact(err).Error()
+	return err.Error()
```

Comment:
is there a difference between this and the `fmt.Sprintf("%+v", err)`?

Reply:
`ErrPanic` will have stacktrace in the error string. I'm supportive with this change.